ADOPTERS.md

```diff
@@ -0,0 +1,11 @@
+The following organizations are known adopters who use Ratify in production or integrate Ratify into their products and services.
+
+We are happy and proud to have you all as part of the Ratify community! To add your use case and organization to this list, please raise a pull request.
+
+All organizations are sorted alphabetically below.
+
+| Adopter name        | Adopter Type     |  Scenario                            |
+|---------------------|------------------|-------------------------------------------|
+| Amazon    | Service provider | [AWS Signer](https://ratify.dev/docs/quickstarts/ratify-with-aws-signer) uses Ratify to verify signature |
+| Microsoft    | Service Provider | [Image Integrity](https://learn.microsoft.com/en-us/azure/aks/image-integrity) is built on Ratify to enable users to validate image signature and supply chain attestations before deploying them to your Azure Kubernetes Service (AKS)  |
+|  Venafi | Service Provider |  [CodeSign Protect](https://venafi.com/codesign-protect/) uses Ratify to help developers maintain visibility into enterprise code signing operations |
```

Comment:
@zosocanuck would you mind reviewing this change?

Reply:
LGTM